pkg/resources/delete_cluster_gce.go

```diff
@@ -400,7 +400,7 @@ func (d *clusterDiscoveryGCE) listGCEDisks() ([]*ResourceTracker, error) {
 		}
 
 		for _, u := range t.Users {
-			tracker.blocked = append(tracker.blocked, typeInstance+":"+t.Zone+"/"+gce.LastComponent(u))
+			tracker.blocked = append(tracker.blocked, typeInstance+":"+gce.LastComponent(t.Zone)+"/"+gce.LastComponent(u))
```

Comment:
If you check https://github.com/google/google-api-go-client/blob/master/compute/v0.beta/compute-gen.go, the `Zone` field in the `Disk` struct is actually a URL and not the zone itself. In the instance resource we expect the zone name and not the zone URL https://github.com/kubernetes/kops/blob/master/pkg/resources/delete_cluster_gce.go#L327.